statements-control/Library/DAOs/ConnectionDB.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Library.DAOs
+{
+    class ConnectionDB
+    {
+        public static SqlConnection GetConexao(string catalog, string userId, string password)
```

Comment:
Nome do método esta em português 😄 
```suggestion
        public static SqlConnection GetConnection(string catalog, string userId, string password)
```